### ISSUE TITLE
Intra-document links do not appear to work

### ISSUE THREAD

**Issue by hctagalong** [fabiospampinato/notable#396]
- **OS Version**: macOS High Sierra 10.13.6
- **Notable Version**: 1.2.0

### Current behavior

Attempting to create a table-of-contents within a note to ease jumping to different headers in the doc. The links are rendered properly, however, these do not jump to the anchor. Here's an example:

```
# Document title

## Table of Contents

* [Heading 1](#heading-1)
* [Heading 2](#heading-2)
* [Heading 3](#heading-3)

## Heading 1

Riveting text here....

## Heading 2

Even more riveting text...

## Heading 3

Less interesting text; didn't save the best for last! :)
```
### Expected behavior

Clicking on the TOC link _should_ jump to the proper header, however, I'm not seeing that behavior in Notable. This does work in VSCode preview.

**Comment by fabiospampinato** [comment]
@hctagalong It's working for me, are you sure the preview is scrollable? (otherwise there's nothing to scroll)

**Comment by hctagalong** [comment]
Yep, the preview is scrollable. I've shrunk the window small enough that 'heading 3' would be scrolled off the page. When I mouse over it I can scroll down (scroller to the right shows, etc.). Clicking the link for 'heading 3' doesn't have any effect.

As part of my test, I select to open in default app (VSCode in my case) and the I can verify that the links work as desired.

**Comment by Fastidious** [comment]
@hctagalong could you share the raw contents of that note (TOC links included)?

**Comment by hctagalong** [comment]
@Fastidious Are you looking for something other than the code block I supplied above?

**Comment by hctagalong** [comment]
Verified -- fixed in v1.3.0.

Incidentally, I wasn't nudged to update notable from v1.2.0. I have it installed as a cask if that makes any difference. Forced the upgrade; all good now.

Appreciate your help!

**Comment by Anzan** [comment]
Anchor links within a document stopped working for me with Notable v 1.8.0 (Windows 10), it worked fine prior to this new version. When I click on it, now it doesn't do anything. This is a very valuable feature for long documents.
